crates/swc_ecma_parser/src/lexer/table.rs

```diff
@@ -59,8 +59,230 @@ const ERR: ByteHandler = Some(|lexer| {
+        "intrinsic" => Some(Word::Ident(IdentLike::Known(KnownIdent::Intrinsic))),
+        _ => None,
+    })
+});
+
+const L_J: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "let" => Some(Word::Keyword(Keyword::Let)),
+        _ => None,
+    })
+});
+
+const L_K: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "keyof" => Some(Word::Ident(IdentLike::Known(KnownIdent::Keyof))),
+        _ => None,
+    })
+});
+
+const L_L: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "let" => Some(Word::Keyword(Keyword::Let)),
+        _ => None,
+    })
+});
+
+const L_M: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "meta" => Some(Word::Ident(IdentLike::Known(KnownIdent::Meta))),
+        _ => None,
+    })
+});
+
+const L_N: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "new" => Some(Word::Keyword(Keyword::New)),
+        "null" => Some(Word::Null),
+        "number" => Some(Word::Ident(IdentLike::Known(KnownIdent::Number))),
+        "never" => Some(Word::Ident(IdentLike::Known(KnownIdent::Never))),
+        "namespace" => Some(Word::Ident(IdentLike::Known(KnownIdent::Namespace))),
+        _ => None,
+    })
+});
+
+const L_O: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "of" => Some(Word::Ident(IdentLike::Known(KnownIdent::Of))),
+        "object" => Some(Word::Ident(IdentLike::Known(KnownIdent::Object))),
+        _ => None,
+    })
+});
+
+const L_P: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "public" => Some(Word::Ident(IdentLike::Known(KnownIdent::Public))),
+        "pacakge" => Some(Word::Ident(IdentLike::Known(KnownIdent::Package))),
+        "protected" => Some(Word::Ident(IdentLike::Known(KnownIdent::Protected))),
+        "private" => Some(Word::Ident(IdentLike::Known(KnownIdent::Private))),
+        _ => None,
+    })
+});
+
+const L_Q: ByteHandler = IDN;
+
+const L_R: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "return" => Some(Word::Keyword(Keyword::Return)),
+        "readonly" => Some(Word::Ident(IdentLike::Known(KnownIdent::Readonly))),
+        "require" => Some(Word::Ident(IdentLike::Known(KnownIdent::Require))),
+        _ => None,
+    })
+});
+
+const L_S: ByteHandler = Some(|lexer| {
+    lexer.read_word_with(|s| match s {
+        "super" => Some(Word::Keyword(Keyword::Super)),
+        "static" => Some(Word::Ident(IdentLike::Known(KnownIdent::Static))),
+        "switch" => Some(Word::Keyword(Keyword::Switch)),
+        "symbol" => Some(Word::Ident(IdentLike::Known(KnownIdent::Symbol))),
+        "set" => Some(Word::Ident(IdentLike::Known(KnownIdent::Set))),
+        "string" => Some(Word::Ident(IdentLike::Known(KnownIdent::String))),
+        "satisfies" => Some(Word::Ident(IdentLike::Known(KnownIdent::Satisfies))),
```

Comment:
Some TypeScript keywords are also included. We did not do this before.
Is that okay?

Reply:
It's intended. It allows skipping an access to `phf::Map` from https://github.com/swc-project/swc/pull/8226/files#diff-6402bc52db5ddbd503b09a01f2ab9ec09c93953a43f40d641aa0edfa83f0cf08R789

We don't check for `KnownIdent`